expansion/src/main/java/no/unit/nva/expansion/model/ExpandedImportCandidate.java

```diff
@@ -313,6 +314,7 @@ private static List<Organization> extractOrganizations(ImportCandidate importCan
         return importCandidate.getEntityDescription().getContributors().stream()
                    .map(Contributor::getAffiliations)
                    .flatMap(List::stream)
+                   .filter(organization -> nonNull(organization.getId()))
```

Comment:
Not sure if all organizations in the import list has an identifier. Ask Terje.

Reply:
Ex. People working in foreign universities.

Reply:
As I can see sentralImport is listing only Norwegian institutions, if we list all the institution we are going to end up with very long list.